### PR TITLE
Port StandardMaterial to webgpu

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15349/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15349/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15349/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15349/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15349/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15349/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15349/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15349/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15349/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15349/merge/testResults/webgpuplaywright/index.html

**Review by sebavan — CHANGES_REQUESTED** [review]
Once [packages/tools/tests/test/visualization/webgpu_webgpu_log.txt](https://github.com/BabylonJS/Babylon.js/pull/15349/files#diff-1d11bde17281f736c21d06cae2be0d288d0637f0d6e1b3d2da67dd56c4cfdcfc) is removed, all good to be merged :-)

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15349/merge/testResults/webgpuplaywright/index.html

**Review by sebavan — APPROVED** [review]

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15349/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15349/merge/testResults/webgpuplaywright/index.html

File: packages/dev/core/src/Materials/standardMaterial.ts [hunk 1]
```diff
@@ -61,6 +59,8 @@ import {
     PrepareUniformsAndSamplersList,
 } from "./materialHelper.functions";
 import { SerializationHelper } from "../Misc/decorators.serialization";
+import { UniformBuffer } from "./uniformBuffer";
```
Comment (RaananW) [inline packages/dev/core/src/Materials/standardMaterial.ts:62]: leaving a comment to avoid merging for now (need to push some changes)

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15349/merge/testResults/webgpuplaywright/index.html

**Comment by deltakosh** [comment]
@Popov72 : I ported almost everything but the OIT PG seems to resist: #1PLV5Z#104

I had to move the depth output to be a vec4 and I don't know if this is the cause of the bad rendering

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15349/merge/testResults/webgpuplaywright/index.html

**Comment by deltakosh** [comment]
@Popov72: I found my mistake :D

**Pull request merged**